apps/asap-server/src/controllers/dashboard.ts

```diff
@@ -0,0 +1,96 @@
+import { GraphQL } from '@asap-hub/services-common';
+import { DashboardResponse } from '@asap-hub/model';
+import {
+  CMSNewsAndEvents,
+  parse as parseNewsAndEvents,
+} from '../entities/news-and-events';
+import { CMSPage, parse as parsePage } from '../entities/page';
+
+export const query = `
```

Comment:
I'm wondering:
If we can query Squidex via GraphQL (even if we end up exposing it as REST from the backend to save implementation time), why did we need to store any redundant data?